### DISCUSSION TITLE
Gluetun Server_Regions issue

### DISCUSSION THREAD

**Discussion by phreatic62** [qdm12/gluetun#1608] [Q&A]
HI,

Hope this is the appropriate place and information required.

Since the last update, my gluetun container stopped working.
I had used Server_Regions to specify 3 preferred regions.

In the log, I get:
=====================================================================
2023-05-26T09:31:29+01:00 ERROR VPN settings: provider settings: server selection: for VPN service provider private internet access: the region specified is not valid: value is not one of the possible choices: value uk london, choices available are AU Australia Streaming Optimized, AU Brisbane, AU Melbourne, AU Perth, AU Sydney, Albania, Algeria, Andorra, Argentina, Armenia, Austria, Bahamas, Bangladesh, Belgium, Bosnia and Herzegovina, Brazil, Bulgaria, CA Montreal, CA Ontario, CA Toronto, CA Vancouver, Cambodia, 
======================================================================

The problem, I think, is that the region is being passed in lower case, when I comment out the Server_Regions line, it works.

**Comment by Z4UBERW4LD** [comment]
i have the same issue since today, using ProtonVPN and for SERVER_COUNTRIES

ERROR VPN settings: provider settings: server selection: for VPN service provider protonvpn: the country specified is not valid: value is not one of the possible choices: value switzerland, choices available are Argentina, Australia, Austria, Belgium, Brazil, Bulgaria, Cambodia, Canada, Chile, Colombia, Costa Rica, Cyprus, Czech Republic, Denmark, Ecuador, Egypt, Estonia, Finland, France, Germany, Greece, Hong Kong, Hungary, Iceland, India, Ireland, Israel, Italy, Japan, Korea, Latvia, Lithuania, Luxembourg, Malaysia, Mexico, Moldova, Netherlands, New Zealand, Nigeria, Norway, Peru, Philippines, Poland, Portugal, Puerto Rico, Romania, Russian Federation, Serbia, Singapore, Slovakia, Slovenia, South Africa, Spain, Sweden, Switzerland, Taiwan, Thailand, Turkey, Ukraine, United Arab Emirates, United Kingdom, United States, Vietnam

**Comment by smtm50** [comment]
Same issue with Privado VPN

ERROR VPN settings: provider settings: server selection: for VPN service provider privado: the country specified is not valid: value is not one of the possible choices: value canada, choices available are Albania, Argentina, Australia, Austria, Brazil, Bulgaria, Canada, Czech Republic, Denmark, Finland, France, Germany, Greece, Hong Kong, Hungary, Iceland, India, Indonesia, Ireland, Israel, Italy, Japan, Korea, Latvia, Malaysia, Netherlands, New Zealand, Norway, Philippines, Poland, Portugal, Romania, Serbia, Singapore, Slovakia, Slovenia, South Africa, Spain, Switzerland, Taiwan, Turkey, United Kingdom, United States

**Reply by frepke** [reply]
Same with Surfshark

**Comment by sunbeam60** [comment]
Yup, same here with Private Internet Access
`2023-05-26T21:38:38+01:00 INFO [routing] local ethernet link found: eth0
2023-05-26T21:38:38+01:00 INFO [routing] local ipnet found: 172.19.0.0/16
2023-05-26T21:38:38+01:00 INFO [firewall] enabling...
2023-05-26T21:38:38+01:00 INFO [firewall] enabled successfully
2023-05-26T21:38:38+01:00 INFO [storage] merging by most recent 13056 hardcoded servers and 13056 servers read f
rom /gluetun/servers.json
2023-05-26T21:38:38+01:00 ERROR VPN settings: provider settings: server selection: for VPN service provider priv
ate internet access: the region specified is not valid: value is not one of the possible choices: value switzerl
and, choices available are AU Australia Streaming Optimized, AU Brisbane, AU Melbourne, AU Perth, AU Sydney, Alb
ania, Algeria, Andorra, Argentina, Armenia, Austria, Bahamas, Bangladesh, Belgium, Bosnia and Herzegovina, Brazi
l, Bulgaria, CA Montreal, CA Ontario, CA Toronto, CA Vancouver, Cambodia, Chile, China, Colombia, Costa Rica, Cr
oatia, Cyprus, Czech Republic, DE Berlin, DE Frankfurt, DK Copenhagen, DK Streaming Optimized, ES Madrid, ES Val
encia, Egypt, Estonia, FI Helsinki, FI Streaming Optimized, France, Georgia, Greece, Greenland, Hong Kong, Hunga
ry, IT Milano, IT Streaming Optimized, Iceland, India, Indonesia, Ireland, Isle of Man, Israel, JP Streaming Opt
imized, JP Tokyo, Kazakhstan, Latvia, Liechtenstein, Lithuania, Luxembourg, Macao, Malaysia, Malta, Mexico, Mold
ova, Monaco, Mongolia, Montenegro, Morocco, Netherlands, New Zealand, Nigeria, North Macedonia, Norway, Panama,
Philippines, Poland, Portugal, Qatar, Romania, SE Stockholm, SE Streaming Optimized, Saudi Arabia, Serbia, Singa
pore, Slovakia, Slovenia, South Africa, Sri Lanka, Switzerland, Taiwan, Turkey, UK London, UK Manchester, UK Sou`

**Comment by phreatic62** [comment]
Seems to be the capitalisation.

**Reply by wpeace3** [reply]
That is what I'm seeing as well, failing on the correct region in all lowercase.

It appears if you remove SERVER_REGION and use SERVER_HOSTNAME it works.
